doc/source/how-to/vulnerabilities.rst

```diff
@@ -0,0 +1,146 @@
+
+Vulnerabilities refer to potential security flaws or weaknesses in PyAnsys packages.
+Here are some examples of common vulnerabilities:
+
+- Insufficient input validation or insecure handling of file paths can lead to unauthorized access
+- Insecure code and command injection into the app
+- Remote code execution due to deserialization of untrusted code
+- Disclosure of sensitive information due to weak exception handling
+
+Vulnerability sources
+---------------------
+
+- **Vulnerabilities from PyAnsys library code**: the maintainers are responsible for deciding whether to
+  address vulnerabilities. The priority of vulnerabilities can be escalated internally if they represent
+  a roadblock for usage. The Ansys business unit in charge of the project should handle
+  vulnerabilities on a case-by-case basis.
+
+- **Vulnerabilities from external package dependencies**: when vulnerabilities exist in external packages
+  used by PyAnsys libraries, such as numpy or matplotlib, Ansys should not address these vulnerabilities
+  directly. Instead, it is best to raise an issue on the open source repository, pointing out the
+  vulnerability and linking the applicable `CVE`_. At most, be mindful of deprecated packages and functions,
+  because they may not receive security patches and might introduce vulnerabilities into your codebase.
+
+Vulnerability discovery and tracking
+-------------------------------------
+
+Leverage available security tools, such as dependency scanners or static
+analyzers (such as PyUp, Safety, or Bandit), to automatically detect and
+remediate security vulnerabilities in Python packages and dependencies.
+
+The PyAnsys ecosystem has implemented automated mechanisms to track and
+report vulnerabilities in the codebase. These tools are intended to be integrated
+into the CI/CD pipeline of the repositories.
+
+The PyAnsys Core team has come up with a GitHub Action that can be used to
+automatically check for vulnerabilities in the codebase. This action is based on
+the following tools:
+
+- `Safety`_ - checks installed dependencies for known security vulnerabilities
+- `Bandit`_ - a tool designed to find common security issues in Python code
+
+Safety addresses external dependencies, while Bandit focuses on the codebase itself.
+Both tools have been integrated into the action ``ansys/actions/check-vulnerabilities``.
+
+.. note::
+
+  The tools selected for the action are not definitive. The action can be modified to include
+  other tools or to use different versions of the tools in the future.
+
+For third-party packages, the PyAnsys Core team has listed a set of excluded advisories by which
+the action will not fail. This is done to avoid false positives and to ensure that the action
+does not block the CI/CD pipeline unnecessarily. You can find the list of excluded advisories
+in `the action's documentation`_.
+
+For potential vulnerabilities in the codebase, repositories can configure `Bandit`_ to ignore specific
```

Comment:
```suggestion
For potential vulnerabilities in the codebase, repositories can configure Bandit to ignore specific
```
You want only the first use to be a link so there aren't excessive links on the page.